src/dotnet-setversion/Program.cs

```diff
@@ -55,7 +64,15 @@ private static int RunRecursive(string version)
 
             foreach (var csprojFile in csprojFiles)
             {
-                SetVersion(version, csprojFile);
+                try
```

Comment:
Since this block of code is repeated, it would be good to factor it out into a separate method.

Reply:
Actually, scratch that, I think this is fine as it is.